### PR TITLE
Added a loading spinner

### PR DESCRIPTION
**Pull request by karolsojko**
When you get on the page it's most of the time empty until the content loads - may take some time - so a good idea would be to add a spinner there.

Sorry for the gif quality but you should get the idea:

untitled

@jkbdbrnwsk - what you say?

### CODE REVIEW

**Comment by theanxy** [comment]
Good idea. Can we reuse any of existing spinner icons from https://github.com/x-team/unleash/blob/develop/app/styles/_icons.scss#L514? You can animate it in CSS

File: app/views/home.html [hunk 1]
```diff
@@ -6,6 +6,8 @@ <h3 class="secondary">“Wherever you are, the path is in front of you.”</h3>
 <section ng-if="user" class="animate-if">
   <h2 class="structural">List of paths:</h2>
 
+  <div ng-show="!allUsers.length" class="initializing-spinner"/>
```
Comment (theanxy) [inline app/views/home.html]: Please close DIV for consistency

**Comment by theanxy** [comment]
As for the CSS animation itself, you can create a reusable mixin in https://github.com/x-team/unleash/blob/develop/app/styles/_mixins.scss in case there’s no such animation yet

**Comment by karolsojko** [comment]
@theanxy fixed :) thanks for help :) 

untitled

**Comment by theanxy** [comment]
Looks good! Added minor fixes:

* fixed indentation (should be 2 chars as per `.editorconfig`)
* removed prefixes as we use grunt-autoprefixer

**Pull request merged**